{"created": 1754028268.55803, "duration": 2.9546759128570557, "exitcode": 1, "root": "/media/kensan/LinuxHDD/ITSM-ITmanagementSystem", "environment": {}, "summary": {"passed": 7, "error": 5, "total": 12, "collected": 18, "deselected": 2}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "tests/e2e/__init__.py", "type": "Package"}, {"nodeid": "tests/e2e/pages/__init__.py", "type": "Package"}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py", "type": "Module"}]}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI", "outcome": "passed", "result": [{"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_homepage_loads", "type": "Function", "lineno": 11}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_navigation_menu", "type": "Function", "lineno": 29}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_incident_creation_form", "type": "Function", "lineno": 50}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_search_functionality", "type": "Function", "lineno": 77}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_responsive_design", "type": "Function", "lineno": 102}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_form_validation", "type": "Function", "lineno": 122, "deselected": true}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_accessibility_features", "type": "Function", "lineno": 162}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_error_handling", "type": "Function", "lineno": 184}]}, {"nodeid": "tests/e2e/test_basic_ui.py", "outcome": "passed", "result": [{"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI", "type": "Class"}]}, {"nodeid": "tests/e2e/__init__.py", "outcome": "passed", "result": [{"nodeid": "tests/e2e/test_basic_ui.py", "type": "Module"}]}, {"nodeid": "tests/e2e/pages/__init__.py", "outcome": "passed", "result": []}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E", "outcome": "passed", "result": [{"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_create_incident_complete_workflow", "type": "Function", "lineno": 14}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_assignment_workflow", "type": "Function", "lineno": 50}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_status_transitions", "type": "Function", "lineno": 71}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_escalation_workflow", "type": "Function", "lineno": 110}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_history_tracking", "type": "Function", "lineno": 131}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_search_and_filter", "type": "Function", "lineno": 158}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_form_validation", "type": "Function", "lineno": 199}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_bulk_operations", "type": "Function", "lineno": 213}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_pagination", "type": "Function", "lineno": 235, "deselected": true}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_edit_workflow", "type": "Function", "lineno": 263}]}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py", "outcome": "passed", "result": [{"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E", "type": "Class"}]}], "tests": [{"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_homepage_loads", "lineno": 11, "outcome": "passed", "keywords": ["test_homepage_loads", "TestBasicUI", "e2e", "test_basic_ui.py", "tests/e2e/__init__.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.03040638200036483, "outcome": "passed"}, "call": {"duration": 0.0011295059994154144, "outcome": "passed"}, "teardown": {"duration": 0.00022909299877937883, "outcome": "passed"}}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_navigation_menu", "lineno": 29, "outcome": "passed", "keywords": ["test_navigation_menu", "TestBasicUI", "e2e", "test_basic_ui.py", "tests/e2e/__init__.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.0004028369985462632, "outcome": "passed"}, "call": {"duration": 0.0012272359999769833, "outcome": "passed"}, "teardown": {"duration": 0.0002344390013604425, "outcome": "passed"}}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_incident_creation_form", "lineno": 50, "outcome": "passed", "keywords": ["test_incident_creation_form", "TestBasicUI", "e2e", "test_basic_ui.py", "tests/e2e/__init__.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.00039221699989866465, "outcome": "passed"}, "call": {"duration": 0.0013041050005995203, "outcome": "passed"}, "teardown": {"duration": 0.0002072569986921735, "outcome": "passed"}}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_search_functionality", "lineno": 77, "outcome": "passed", "keywords": ["test_search_functionality", "TestBasicUI", "e2e", "test_basic_ui.py", "tests/e2e/__init__.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.00038846900133648887, "outcome": "passed"}, "call": {"duration": 0.001612963002116885, "outcome": "passed"}, "teardown": {"duration": 0.0002333349984837696, "outcome": "passed"}}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_responsive_design", "lineno": 102, "outcome": "passed", "keywords": ["test_responsive_design", "TestBasicUI", "e2e", "test_basic_ui.py", "tests/e2e/__init__.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.00038431099892477505, "outcome": "passed"}, "call": {"duration": 0.0007946120022097602, "outcome": "passed"}, "teardown": {"duration": 0.00023260799935087562, "outcome": "passed"}}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_accessibility_features", "lineno": 162, "outcome": "passed", "keywords": ["test_accessibility_features", "TestBasicUI", "e2e", "test_basic_ui.py", "tests/e2e/__init__.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.00035520999881555326, "outcome": "passed"}, "call": {"duration": 0.0008786229991528671, "outcome": "passed"}, "teardown": {"duration": 0.00019174599947291426, "outcome": "passed"}}, {"nodeid": "tests/e2e/test_basic_ui.py::TestBasicUI::test_error_handling", "lineno": 184, "outcome": "passed", "keywords": ["test_error_handling", "TestBasicUI", "e2e", "test_basic_ui.py", "tests/e2e/__init__.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.0003459550025581848, "outcome": "passed"}, "call": {"duration": 0.0009476279992668424, "outcome": "passed"}, "teardown": {"duration": 0.0002084440020553302, "outcome": "passed"}}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_create_incident_complete_workflow", "lineno": 14, "outcome": "error", "keywords": ["test_create_incident_complete_workflow", "TestIncidentWorkflowE2E", "e2e_critical", "incidents", "e2e", "tests/e2e/scenarios/test_incident_workflow.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 1.6349462840007618, "outcome": "failed", "crash": {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "traceback": [{"path": "tests/e2e/conftest.py", "lineno": 66, "message": "in authenticated_page"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py", "lineno": 9312, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py", "lineno": 475, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py", "lineno": 139, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 62, "message": "in send"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 492, "message": "in wrap_api_call"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "in inner_send"}], "longrepr": "tests/e2e/conftest.py:66: in authenticated_page\n    page.goto(f\"{playwright_config['base_url']}/login\")\n/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py:9312: in goto\n    self._sync(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py:475: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py:139: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:62: in send\n    return await self._connection.wrap_api_call(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:492: in wrap_api_call\n    return await cb()\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:100: in inner_send\n    result = next(iter(done)).result()\nE   playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "teardown": {"duration": 0.018298197999683907, "outcome": "passed"}}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_assignment_workflow", "lineno": 50, "outcome": "error", "keywords": ["test_incident_assignment_workflow", "TestIncidentWorkflowE2E", "e2e_critical", "incidents", "e2e", "tests/e2e/scenarios/test_incident_workflow.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.07738521699866396, "outcome": "failed", "crash": {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "traceback": [{"path": "tests/e2e/conftest.py", "lineno": 66, "message": "in authenticated_page"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py", "lineno": 9312, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py", "lineno": 475, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py", "lineno": 139, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 62, "message": "in send"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 492, "message": "in wrap_api_call"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "in inner_send"}], "longrepr": "tests/e2e/conftest.py:66: in authenticated_page\n    page.goto(f\"{playwright_config['base_url']}/login\")\n/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py:9312: in goto\n    self._sync(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py:475: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py:139: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:62: in send\n    return await self._connection.wrap_api_call(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:492: in wrap_api_call\n    return await cb()\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:100: in inner_send\n    result = next(iter(done)).result()\nE   playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "teardown": {"duration": 0.014252401997509878, "outcome": "passed"}}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_status_transitions", "lineno": 71, "outcome": "error", "keywords": ["test_incident_status_transitions", "TestIncidentWorkflowE2E", "e2e_critical", "incidents", "e2e", "tests/e2e/scenarios/test_incident_workflow.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.07175323300180025, "outcome": "failed", "crash": {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "traceback": [{"path": "tests/e2e/conftest.py", "lineno": 66, "message": "in authenticated_page"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py", "lineno": 9312, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py", "lineno": 475, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py", "lineno": 139, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 62, "message": "in send"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 492, "message": "in wrap_api_call"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "in inner_send"}], "longrepr": "tests/e2e/conftest.py:66: in authenticated_page\n    page.goto(f\"{playwright_config['base_url']}/login\")\n/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py:9312: in goto\n    self._sync(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py:475: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py:139: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:62: in send\n    return await self._connection.wrap_api_call(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:492: in wrap_api_call\n    return await cb()\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:100: in inner_send\n    result = next(iter(done)).result()\nE   playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "teardown": {"duration": 0.014479267003480345, "outcome": "passed"}}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_escalation_workflow", "lineno": 110, "outcome": "error", "keywords": ["test_incident_escalation_workflow", "TestIncidentWorkflowE2E", "e2e_critical", "incidents", "e2e", "tests/e2e/scenarios/test_incident_workflow.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.07310802600113675, "outcome": "failed", "crash": {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "traceback": [{"path": "tests/e2e/conftest.py", "lineno": 66, "message": "in authenticated_page"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py", "lineno": 9312, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py", "lineno": 475, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py", "lineno": 139, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 62, "message": "in send"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 492, "message": "in wrap_api_call"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "in inner_send"}], "longrepr": "tests/e2e/conftest.py:66: in authenticated_page\n    page.goto(f\"{playwright_config['base_url']}/login\")\n/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py:9312: in goto\n    self._sync(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py:475: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py:139: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:62: in send\n    return await self._connection.wrap_api_call(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:492: in wrap_api_call\n    return await cb()\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:100: in inner_send\n    result = next(iter(done)).result()\nE   playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "teardown": {"duration": 0.01520684799834271, "outcome": "passed"}}, {"nodeid": "tests/e2e/scenarios/test_incident_workflow.py::TestIncidentWorkflowE2E::test_incident_history_tracking", "lineno": 131, "outcome": "error", "keywords": ["test_incident_history_tracking", "TestIncidentWorkflowE2E", "e2e_critical", "incidents", "e2e", "tests/e2e/scenarios/test_incident_workflow.py", "ITSM-ITmanagementSystem"], "setup": {"duration": 0.07037140799729968, "outcome": "failed", "crash": {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "traceback": [{"path": "tests/e2e/conftest.py", "lineno": 66, "message": "in authenticated_page"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py", "lineno": 9312, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py", "lineno": 475, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py", "lineno": 139, "message": "in goto"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 62, "message": "in send"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 492, "message": "in wrap_api_call"}, {"path": "/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py", "lineno": 100, "message": "in inner_send"}], "longrepr": "tests/e2e/conftest.py:66: in authenticated_page\n    page.goto(f\"{playwright_config['base_url']}/login\")\n/home/kensan/.local/lib/python3.12/site-packages/playwright/sync_api/_generated.py:9312: in goto\n    self._sync(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_page.py:475: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_frame.py:139: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:62: in send\n    return await self._connection.wrap_api_call(\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:492: in wrap_api_call\n    return await cb()\n/home/kensan/.local/lib/python3.12/site-packages/playwright/_impl/_connection.py:100: in inner_send\n    result = next(iter(done)).result()\nE   playwright._impl._errors.Error: net::ERR_NAME_NOT_RESOLVED at https://itsm-system.com/login"}, "teardown": {"duration": 0.016574132998357527, "outcome": "passed"}}], "warnings": [{"message": "Unknown pytest.mark.e2e - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/media/kensan/LinuxHDD/ITSM-ITmanagementSystem/tests/e2e/test_basic_ui.py", "lineno": 8}, {"message": "Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/media/kensan/LinuxHDD/ITSM-ITmanagementSystem/tests/e2e/test_basic_ui.py", "lineno": 123}, {"message": "Unknown pytest.mark.e2e - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/media/kensan/LinuxHDD/ITSM-ITmanagementSystem/tests/e2e/scenarios/test_incident_workflow.py", "lineno": 9}, {"message": "Unknown pytest.mark.incidents - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/media/kensan/LinuxHDD/ITSM-ITmanagementSystem/tests/e2e/scenarios/test_incident_workflow.py", "lineno": 10}, {"message": "Unknown pytest.mark.slow - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html", "category": "PytestUnknownMarkWarning", "when": "collect", "filename": "/media/kensan/LinuxHDD/ITSM-ITmanagementSystem/tests/e2e/scenarios/test_incident_workflow.py", "lineno": 236}]}